apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -320,6 +335,32 @@ const CurriculumCatalog = ({curriculaData, ...props}) => {
           color={Button.ButtonColor.brandSecondaryDefault}
         />
       </div>
+      {!isEnglish && (
+        <div className={style.catalogLanguageFilterRow}>
+          <div className={style.catalogLanguageFilterRowNumAvailable}>
+            <BodyOneText>
+              {i18n.numCurriculaAvailableInLanguage({
```

Comment:
I came here to check that this was going to get translated eventually, thank you! 😺